src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+import { MoviesList } from '../MoviesList';
 
 export const FindMovie: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie>();
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(false);
+
+  const loadData = () => {
+    setIsLoading(true);
+
+    return getMovie(query)
+      .then(data => {
+        if ('Error' in data) {
+          setErrorMessage(true);
+        }
```

Comment:
You do not need this cause you have a catch block

Reply:
👌, I understood.